### DISCUSSION TITLE
stayinhull NPCs don't care about crime

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15065] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

https://github.com/user-attachments/assets/ed8a81b3-571e-455e-ab3b-fe80d96687a7

### Reproduction steps

.

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by A2ur3** [comment]
you can also hide behind other crews and shot security officer too

**Comment by itchyOwl** [comment]
Duplicate of #13960. Closing.
